lib/sleep-lib.robot

```diff
@@ -134,9 +138,9 @@ Perform Warmboot Using Rtcwake
     # would hang here and fail.
     # Sometimes it may take long to shutdown all systemd services,
     # so the waiting times have to be excessive to avoid false negatives.
-    Write Into Terminal    rtcwake -m off -s 20
+    Write Into Terminal    rtcwake -m off -s 60
```

Comment:
If the shutdown took more than 20s then the device would not turn on again. It still won't if the shutdown takes more than 60s so that is not a perfect solution...